quiche/src/recovery/congestion/bbr/mod.rs

```diff
@@ -34,13 +34,13 @@ use crate::recovery::*;
 
 use std::time::Duration;
 
-pub static BBR: CongestionControlOps = CongestionControlOps {
+use super::CongestionControlOps;
+
+pub(crate) static BBR: CongestionControlOps = CongestionControlOps {
     on_init,
-    reset,
     on_packet_sent,
     on_packets_acked,
     congestion_event,
-    collapse_cwnd,
```

Comment:
Why reset() and collape_cwnd() is removed? (I understand it's not used, but no plan to implement persistent congestion in the future?)

Reply:
We'll probably want to do it at some point... but that's been the case for years, so for now removing them simplifies the refactoring a bit. We can add them back when needed.